CMakeLists.txt

```diff
@@ -32,16 +36,15 @@ include(KDECMakeSettings)
 include(FeatureSummary)
 
 find_package(
-    Qt5
-    5.15
-    CONFIG
-    REQUIRED Core Gui Widgets Xml PrintSupport LinguistTools
+    Qt6
+    REQUIRED
+    COMPONENTS Core Gui Widgets Xml PrintSupport LinguistTools Core5Compat
```

Comment:
Should the `QT_MIN_VERSION` variable be used here? Again, the `CMakeLists.txt` for konsole is probably a good inspiration. :D